src/modules/dashboard/components/EditColonyDetailsDialog/EditColonyDetailsDialogForm.tsx

```diff
@@ -0,0 +1,205 @@
+import { useAsyncFunction, useTransformer } from '~utils/hooks';
+
+import { getAllUserRoles } from '../../../transformers';
+import { hasRoot } from '../../../users/checks';
+
+import { FormValues } from './EditColonyDetailsDialog';
+import styles from './EditColonyDetailsDialogForm.css';
+
+const MSG = defineMessages({
+  title: {
+    id: 'dashboard.EditColonyDetailsDialog.EditColonyDetailsDialogForm.title',
+    defaultMessage: 'Edit colony details',
+  },
+  name: {
+    id: 'dashboard.EditColonyDetailsDialog.EditColonyDetailsDialogForm.name',
+    defaultMessage: 'Colony name',
+  },
+  logo: {
+    id: 'dashboard.EditColonyDetailsDialog.EditColonyDetailsDialogForm.logo',
+    defaultMessage: 'Colony Logo (Optional)',
+  },
+  permittedFormat: {
+    id:
+      // eslint-disable-next-line max-len
+      'dashboard.EditColonyDetailsDialog.EditColonyDetailsDialogForm.permittedFormat',
+    defaultMessage:
+      'Permitted format: .png or .svg (at least 250px, up to 1MB)',
+  },
+  annotation: {
+    id:
+      // eslint-disable-next-line max-len
+      'dashboard.EditColonyDetailsDialog.EditColonyDetailsDialogForm.annotation',
+    defaultMessage: `Explain why you’re editing the colony's details (optional)`,
+  },
+  noPermission: {
+    id:
+      // eslint-disable-next-line max-len
+      'dashboard.EditColonyDetailsDialog.EditColonyDetailsDialogForm.noPermission',
+    defaultMessage: `You do not have the {roleRequired} permission required
+      to take this action.`,
+  },
+});
+
+const ColonyAvatar = HookedColonyAvatar({ fetchColony: true });
+
+interface Props {
+  back: () => void;
+  colony: Colony;
+}
+
+const uploadActions = {
+  submit: ActionTypes.COLONY_AVATAR_UPLOAD,
+  success: ActionTypes.COLONY_AVATAR_UPLOAD_SUCCESS,
+  error: ActionTypes.COLONY_AVATAR_UPLOAD_ERROR,
+};
+
+const removeActions = {
+  submit: ActionTypes.COLONY_AVATAR_REMOVE,
+  success: ActionTypes.COLONY_AVATAR_REMOVE_SUCCESS,
+  error: ActionTypes.COLONY_AVATAR_REMOVE_ERROR,
+};
+
+const EditColonyDetailsDialogForm = ({
+  back,
+  colony,
+  colony: { colonyAddress, avatarHash },
+  handleSubmit,
+  isSubmitting,
+}: Props & FormikProps<FormValues>) => {
+  const { walletAddress, username, ethereal } = useLoggedInUser();
+
+  const allUserRoles = useTransformer(getAllUserRoles, [colony, walletAddress]);
+
+  const hasRegisteredProfile = !!username && !ethereal;
+
+  const userHasPermission = hasRegisteredProfile && hasRoot(allUserRoles);
+
+  const transform = useCallback(
+    pipe(withKey(colonyAddress), mergePayload({ colonyAddress })),
+    [colonyAddress],
+  );
+
+  const upload = useAsyncFunction({ ...uploadActions, transform }) as any;
+  const remove = useAsyncFunction({ ...removeActions, transform }) as any;
+
+  return (
+    <>
+      <DialogSection appearance={{ theme: 'heading' }}>
+        <Heading
+          appearance={{ size: 'medium', margin: 'none' }}
+          text={MSG.title}
+          className={styles.title}
+        />
```

Comment:
This heading had a different color for the other ones. Please change it to be more dark (see the one below)

![Screenshot from 2021-01-19 13-49-29](https://user-images.githubusercontent.com/1193222/105033010-3788bb00-5a60-11eb-84a7-fbdc62ffb6be.png)